lib/spree/chimpy/interface/customer_upserter.rb

```diff
@@ -60,28 +58,6 @@ def upsert_customer
         end
         customer_id
       end
-
-      def upsert_customer_merge_vars
```

Comment:
not sure you want to remove this, this gets the merge values and does subscribe...the below method transforms